.storybook/config.js

```diff
@@ -1,7 +1,7 @@
 import { configure } from '@kadira/storybook';
 
 function loadStories() {
-  require('../webpack/stories');
+  require('../webpack/react_app/storybook');
```

Comment:
Since it includes react files, and is dependant on react, it made sense to me to put it under react_app